client/state/site-settings/test/reducer.js

```diff
@@ -0,0 +1,302 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SITE_SETTINGS_RECEIVE,
+	SITE_SETTINGS_REQUEST,
+	SITE_SETTINGS_REQUEST_FAILURE,
+	SITE_SETTINGS_REQUEST_SUCCESS,
+	SITE_SETTINGS_SAVE,
+	SITE_SETTINGS_SAVE_FAILURE,
+	SITE_SETTINGS_SAVE_SUCCESS,
+	SITE_SETTINGS_UPDATE,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+import { items, requesting, saving } from '../reducer';
+
+describe( 'reducer', () => {
+	before( () => {
```

Comment:
See an example here of how I updated a test to simplify this and avoid need for disabling ESLint rule:

https://github.com/Automattic/wp-calypso/pull/9039/files#diff-17